src/views/virtualmachinesinstance/details/tabs/console/components/vnc-console/utils/VNCConsoleConsts.ts

```diff
@@ -0,0 +1,6 @@
+export enum VNCState {
+  'connecting' = 'connecting',
```

Comment:
shouldn't this be without the single quotes?

Reply:
without single quote is:
```js
export enum VNCState {
  connecting,
}
```

Reply:
actually enum in typescript in this way:
export enum VNCState {
  connecting,
dissocnnected
}

will provide values of connecting = 0 , disconnected = 1 , which is a number, the VNCState is string base, so I cannot use the short version.